config/site_settings.yml

```diff
@@ -280,7 +280,7 @@ basic:
     default: true
   enable_badge_sql:
     client: true
-    default: false
+    default: true
```

Comment:
Was this enabled by accident? It doesn't seem relevant for this PR.

Reply:
Yes, thanks for catching it!